src/__tests__/credentials-autoAccept.test.ts

```diff
@@ -0,0 +1,874 @@
+import type { Agent } from '../agent/Agent'
```

Comment:
I think it would be nice if we can remove some of the bloat from this file. `credentials.test.ts` does a lot of checks. I think with auto-accept we can remove a lot of those checks and only test what we're really testing: auto-acceptance. This makes it easier to maintain this test over time